### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by ks-rahul**
CAN-254
CAN-255
CAN-258
CAN-292
CAN-293
CAN-296
CAN-298
CAN-299
CAN-300

### CODE REVIEW

File: src/components/ComponentPages/ForgotPassword/UI/index.tsx [hunk 1]
```diff
@@ -57,8 +57,8 @@ function ForgotPasswordUI({ form, onFinish, isModal, closeModal, isScreen }) {
           )}
           {isScreen === 1 && (
             <Text type="danger" className={styles.otpNote}>
-              Note : ForgotPassword code has been sent to your registered email
-              address and Phone Number.
+              Note : OTP has been sent to your registered email address and
```
Comment (akuks) [inline src/components/ComponentPages/ForgotPassword/UI/index.tsx]: Please move it to messages

Comment (akuks) [reply src/components/ComponentPages/ForgotPassword/UI/index.tsx]: I mean the message

File: src/components/ComponentPages/Form/ProfileInfoForm.tsx [hunk 2]
```diff
@@ -93,13 +91,13 @@ function ProfileInfoForm({
         {suggestions.map((suggestion, index) => {
           const style = suggestion.active
             ? {
-              backgroundColor: "#f8f8f8",
-              cursor: "pointer",
-            }
+                backgroundColor: "#f8f8f8",
```
Comment (akuks) [inline src/components/ComponentPages/Form/ProfileInfoForm.tsx]: Can you please move it to common place and use it like `colors.profile.background` ?

File: src/components/ComponentPages/Form/ProfileInfoForm.tsx [hunk 3]
```diff
@@ -93,13 +91,13 @@ function ProfileInfoForm({
...
+                cursor: "pointer",
+              }
             : {
-              backgroundColor: "#ffffff",
-              cursor: "pointer",
-            };
+                backgroundColor: "#ffffff",
```
Comment (akuks) [inline src/components/ComponentPages/Form/ProfileInfoForm.tsx]: Can you please move it to common place and use it like `colors.profile.background` ?

**Review by akuks — CHANGES_REQUESTED** [review]
Please do the respective changes

File: src/components/ComponentPages/Form/ProfileInfoForm.tsx (hunk 3 above)
Comment (ks-rahul) [reply src/components/ComponentPages/Form/ProfileInfoForm.tsx]: this section done by Vikas

Comment (ks-rahul) [reply src/components/ComponentPages/Form/ProfileInfoForm.tsx]: Vikas has worked on this component

File: src/components/ComponentPages/ForgotPassword/UI/index.tsx (hunk 1 above)
Comment (ks-rahul) [reply src/components/ComponentPages/ForgotPassword/UI/index.tsx]: moved

File: src/components/ComponentPages/Form/ProfileInfoForm.tsx (hunk 2 above)
Comment (ks-rahul) [reply src/components/ComponentPages/Form/ProfileInfoForm.tsx]: Vikas has worked on this component

**Pull request merged**